rosters.py

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env python3
+
+import argparse
+import csv
+import os
+import re
+import requests
+from getpass import getpass
+
+# ANSI color codes
+GREEN = "\033[01;92m"
+RED = "\033[01;91m"
+END = "\033[0m"
+
+def download_roster(course_id, canvas_api_key):
+    response = requests.get(f'https://courseworks2.columbia.edu/api/v1/courses/{course_id}/students',
+            headers={'Authorization': f'Bearer {canvas_api_key}'}).json()
+
+    response = list(filter(lambda x: x['sortable_name'] != 'Student, Test', response))
```

Comment:
I'm not a big fan of the `list(filter())` style, would prefer a conditional list comprehension instead, but up to you. Same for the instances below